crates/bevy_picking/src/hover.rs

```diff
@@ -275,3 +275,266 @@ fn merge_interaction_states(
         new_interaction_state.insert(*hovered_entity, new_interaction);
     }
 }
+
+/// A component that allows users to use regular Bevy change detection to determine when the pointer
+/// enters or leaves an entity. Users should insert this component on an entity to indicate interest
+/// in knowing about hover state changes.
+///
+/// This is similar to the old Bevy [`Interaction`] component, except that it only tracks
```

Comment:
This feels a bit awkward as "old" implies Interaction either no longer exists or is deprecated. Otherwise this is just a history lesson about what came first, which isn't really something users need to care about.